.github/workflows/release.yml

```diff
@@ -22,8 +22,12 @@ jobs:
 
   release:
     name: Release
-    runs-on: ubuntu-18.04 
+    runs-on: macos-10.15
     needs: [authorize]
+    strategy:
+      matrix:
```

Comment:
curious, why use matrix if we are only using a single version for ruby and node?

Reply:
Do this because for ruby there is more than 1 place we need to call the ruby version. I want to make it easier to reference. If we want to change the ruby version, we only need to change one place.